src/ert/analysis/_es_update.py

```diff
@@ -387,6 +342,10 @@ def _copy_unupdated_parameters(
 
     # Copy the non-updated parameter groups from source to target for each active realization
     for parameter_group in not_updated_parameter_groups:
+        if source_ensemble.has_combined_parameter_dataset(parameter_group):
+            unified_ds = source_ensemble.load_parameters(parameter_group)
+            target_ensemble.save_parameters_combined(parameter_group, unified_ds)
```

Comment:
Seems like there is missing an `else` here? And maybe a test. Do we need this? Or better to just loop? This is a quite rare occurrence (at least presently), not not much gained by optimizing.

Reply:
The else was missing indeed.
Wrt the optimization, we'd have `num_reals*num_param_groups` small .nc files created, before being combined to a file identical to the combined file we pulled them out of. Though I'm also missing some context on how much/often this happens.

Reply:
This is quite rare, though might see more use in the future, if it is an optimization I suggest dropping it for now and simplifying instead.